src/custom-elements/src/components/time/time.tsx

```diff
@@ -226,10 +260,10 @@ export class Time {
       let minuteState = '';
 
       if (parseInt(this._minute) === parseInt(minute)) {
-        minuteState = '-active';
+        minuteState = ACTIVE_CLASS;
       } else {
         if (this.excludedMinutesArray.includes(minute)) {
-          minuteState += ' -disabled';
+          minuteState += ` ${DISABLED_CLASS}`;
```

Comment:
Why the space?

Reply:
Good catch. I believe the space and += were leftover parts from the first approach that has been changed. Removed 👍